Source/WebCore/platform/graphics/gstreamer/eme/GStreamerEMEUtilities.h

```diff
@@ -87,13 +88,9 @@ class ProtectionSystemEvents {
         for (unsigned i = 0; i < numEvents; ++i)
             m_events.append(GRefPtr<GstEvent>(static_cast<GstEvent*>(g_value_get_boxed(gst_value_list_get_value(streamEncryptionEventsList, i)))));
         const GValue* streamEncryptionAllowedSystemsValue = gst_structure_get_value(structure, "available-stream-encryption-systems");
-        WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port
-        const char** streamEncryptionAllowedSystems = reinterpret_cast<const char**>(g_value_get_boxed(streamEncryptionAllowedSystemsValue));
-        WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
-        if (streamEncryptionAllowedSystems) {
-            for (unsigned i = 0; streamEncryptionAllowedSystems[i]; ++i)
-                m_availableSystems.append(String::fromLatin1(streamEncryptionAllowedSystems[i]));
-        }
+        auto streamEncryptionAllowedSystems = span(static_cast<char**>(g_value_get_boxed(streamEncryptionAllowedSystemsValue)));
```

Comment:
I understand this would be the overload that `span` would take here:

```
static inline std::span<char*> span(char** strv)
{
    auto size = g_strv_length(strv);
    return unsafeMakeSpan(strv, size);
}
```

However, according to [g_strv_length](https://docs.gtk.org/glib/func.strv_length.html), `str_array` should not be null and the previous code accounted for a possibly null value out of [g_value_get_boxed](https://docs.gtk.org/gobject/method.Value.get_boxed.html) (which the documentation states as a possibility). 

Do you think this is safe?

Reply:
Should be, but a nullptr check wouldn't hurt :)